Public/Src/Tools/DropDaemon/Program.cs

```diff
@@ -950,66 +939,5 @@ internal static string ToPayload(string commandName, Config config)
         private static T RegisterDaemonConfigOption<T>(T option) where T : Option => RegisterOption(DaemonConfigOptions, option);
 
         private static T RegisterDropConfigOption<T>(T option) where T : Option => RegisterOption(DropConfigOptions, option);
-
-        private static int SubscribeAndProcessCloudBuildEvents()

```

Comment:
This is safe to remove, since it is never actually used for anything.  It was used for some testing at one point for a short period of time.